dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -600,3 +606,48 @@ def validate_job(self, request, interfaces):
                 ValidationError.unknownError)
             self.markJob(job_id, jobTracker, "failed", self.filename, ValidationError.unknownError)
             return JsonResponse.error(exc, exc.status)
+
+
+def update_tas_ids(model, submission_id):
+    sess = GlobalDB.db().session
+    submission = sess.query(Submission).\
+        filter_by(submission_id=submission_id).one()
+
+    # Due to the OVERLAPS, tuples, and IS NOT DISTINCT FROM, this query is
+    # more gnarly in sqlalchemy than straight SQL, so using SQL instead
+    sql = """
+        UPDATE {table_name}
+        SET tas_id = (
+            SELECT min(tas.tas_id)
```

Comment:
From what I understand, it's technically possible, but shouldn't come up in practice. I'll document in a note to see what you think.